### PR TITLE
WRO-3172: Updated agate samples with new React style APIs

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Converted class components into functional components in agate samples

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-3172

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: samples/sampler/stories/qa/Dropdown.js [hunk 1]
```diff
@@ -18,65 +19,51 @@ const list = [
...
-					{['a', 'b', 'c']}
-				</Dropdown>
-			</div>
-		);
-	}
-}
+	return (
+		<div>
+			<Button onClick={handleClick}>{isDisabled ? 'Enable dropdown' : 'Disable dropdown'}</Button>
+			<Dropdown title="hello" disabled={isDisabled}>
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/qa/Dropdown.js:61]: Can you sort the props please?

File: samples/sampler/stories/qa/Panels.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 /* eslint-disable react/jsx-no-bind */
+import Header from '@enact/agate/Header';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import {Panels, Panel, BreadcrumbPanels} from '@enact/agate/Panels';
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/qa/Panels.js:5]: Can you sort the components import please?

File: samples/sampler/stories/qa/Panels.js [hunk 3]
```diff
@@ -121,36 +121,31 @@ const BreadcrumbPanelsBase  = kind({
...
-					<Route path="settings" component={MainPanel}>
-						<Route path="page1" component={Page1}>
-							<Route path="endPage" component={EndPage} />
-						</Route>
-						<Route path="page2" component={Page2}>
-							<Route path="endPage" component={EndPage} />
-						</Route>
+	return (
+		<div {...props}>
+			<RoutablePanels path={appPath} onNavigate={onNavigate} cover="partial">
```
Comment (vladut-clapou-lgp) [inline samples/sampler/stories/qa/Panels.js:133]: Can you sort the props from line 133 to 142 please?

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

File: samples/qa-a11y/src/views/ReadAlert.js [hunk 4]
```diff
@@ -25,61 +22,53 @@ class ReadAlertView extends Component {
...
+			<Section title="AudioGuidance On or Off">
+				<CheckboxItem
+					alt="Toggle"
+					defaultSelected={audioGuidance}
+					disabled={toggleDisabled}
+					onToggle={onToggle}
+				>
+					Audio guidance
+				</CheckboxItem>
+			</Section>
```
Comment (stanca-pop-lgp) [inline samples/qa-a11y/src/views/ReadAlert.js:65]: Please remove this empty line.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**